python/lsst/afw/table/ampInfo.cc

```diff
@@ -34,16 +36,112 @@ using namespace pybind11::literals;
 namespace lsst {
 namespace afw {
 namespace table {
+namespace {
+
+using PyAmpInfoRecord = py::class_<AmpInfoRecord, std::shared_ptr<AmpInfoRecord>, BaseRecord>;
+using PyAmpInfoTable = py::class_<AmpInfoTable, std::shared_ptr<AmpInfoTable>, BaseTable>;
+using PyAmpInfoCatalog =  py::class_<CatalogT<AmpInfoRecord>, std::shared_ptr<AmpInfoCatalog>>;
+
+void declareAmpInfoRecord(PyAmpInfoRecord & cls) {
+    table::pybind11::addCastFrom<BaseRecord, AmpInfoRecord>(cls);
+
+    cls.def("getName", &AmpInfoRecord::getName);
+    cls.def("setName", &AmpInfoRecord::setName, "name"_a,
+                         "Set name of amplifier location in camera");
```

Comment:
Docstrings? I thought we didn't do that yet? I don't mind, but I just want to avoid setting a precedent.